util/put_text.lua

```diff
@@ -0,0 +1,53 @@
+--- Set of 2 functions for throwing text at the end / start of a line from normal mode
+--- It doesn't move you from your current position and doesn't populate Vim marks
+---
+--- Usage: It accepts both single and multi-char inputs, including spaces.
+--- vim.keymap.set("n", "<leader>,", put_at_end(","))
+--- vim.keymap.set("n", "<leader>-", put_at_beginning("- "))
+--- vim.keymap.set("n", "<leader>{", put_at_beginning("{"))
+--- vim.keymap.set("n", "<leader>}", put_at_end("]"))
```

Comment:
These examples don't work as they're written out here. The calls to `put_at_end` and `put_at_beginning` need to be wrapped in a function.

I also believe that the 4th example was supposed to be `}` instead of `]`?

```suggestion
---
---   vim.keymap.set("n", "<leader>,", function()
---     put_at_beginning(",")
---   end)
---
---   vim.keymap.set("n", "<leader>-", function()
---     put_at_beginning("- ")
---   end)
---
---   vim.keymap.set("n", "<leader>{", function()
---     put_at_beginning("{")
---   end)
---
---   vim.keymap.set("n", "<leader>}", function()
---     put_at_end("}")
---   end)
---
```